### PR TITLE
chore(release): 4.8.0 release notes

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

release notes for [4.8.0  milestone](https://github.com/medic/cht-core/milestone/128?closed=1)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: content/en/core/releases/4.8.0.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+The Contacts page loads much faster now. Using [apdex scores and testing on low spec devices](https://forum.communityhealthtoolkit.org/t/evaluating-app-performance-using-apdex-lets-dive-in-and-look-at-an-example/3475), we were able to identify slow areas and make improvements to both the [list view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-list-view/3501) and the [detail view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-detail-view/3486). 
+
+
+- [#9006](https://github.com/medic/cht-core/issues/9006): Improves performance of the list view by reducing the number of rows fetched each time
+- [#9019](https://github.com/medic/cht-core/issues/9019): Improves performance of the detail view by reducing loops for tasks and reports
+
+
+### Security
+
+Three  security issues with severity level of "high" were fixed in this release.  While Medic strives to have vulnerability free releases, we're always keeping an eye out for any issues that may have slipped through.  Thanks to these ongoing efforts, the three security issues fixed in this release are:
```
Comment (garethbowen) [inline content/en/core/releases/4.8.0.md:39]:
```suggestion
Three long standing security issues were fixed in this release, which were detected during routine penetration tests. We recommend updating as soon as possible to get the best possible cybersecurity protection.
```

File: content/en/core/releases/4.8.0.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+* Protect against a malicious 3rd party using the redirect after login feature to steal credentials
+* Ensure users with basic access are not allowed to self-promote themselves to a user with more permissions
+* Ensure users with basic access are not allowed to sensitive configuration files, like translations
+
+For more details on these issue, please see the relevant tickets:
+
+- [#8886](https://github.com/medic/cht-core/issues/8886): Block users from editing important fields in user settings docs
+- [#9108](https://github.com/medic/cht-core/issues/9108): Admin only docs can be modified by non-admin users
+- [#9122](https://github.com/medic/cht-core/issues/9122): Protect against redirection attack
```
Comment (garethbowen) [inline content/en/core/releases/4.8.0.md:49]: This is redundant - merge these two lists into one.

File: content/en/core/releases/4.8.0.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+None.
+
+### Improvements
+
+- [#8884](https://github.com/medic/cht-core/issues/8884): Add Swahili Translations
+- [#9118](https://github.com/medic/cht-core/issues/9118): Use helm repository for cht-core helm-chart
+
+### Security fixes
+
+- [#8338](https://github.com/medic/cht-core/issues/8338): Research increasing pbkdf2 iterations
```
Comment (garethbowen) [inline content/en/core/releases/4.8.0.md:64]: This one got moved to 4.9.0 so should be removed from here.

```suggestion
```

File: content/en/core/releases/4.8.0.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+### UI/UX changes
+
+None.
+
+
+## Highlights
+
+### Performance
+
+The Contacts page loads much faster now. Using [apdex scores and testing on low spec devices](https://forum.communityhealthtoolkit.org/t/evaluating-app-performance-using-apdex-lets-dive-in-and-look-at-an-example/3475), we were able to identify slow areas and make improvements to both the [list view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-list-view/3501) and the [detail view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-detail-view/3486).
```
Comment (garethbowen) [inline content/en/core/releases/4.8.0.md:30]: Add a graph showing apdex score before and after using the tables on those forum posts. It's maybe not the most exciting graph but anything visual helps make the release notes more interesting.

File: content/en/core/releases/_index.md [hunk 5]
```diff
@@ -22,7 +22,8 @@ It is recommended that all projects update regularly multiple times a year to ge
 
 | Version | Status    | Release date | End of life |
 |---------|-----------|--------------|-------------|
-| 4.7.x   | Current   | 07-May-2024  | TBA         |
+| 4.8.x   | Current   | 22-May-2024  | TBA         |
+| 4.7.x   | Supported | 07-May-2024  | 10-Oct-2024 |
```
Comment (garethbowen) [inline content/en/core/releases/_index.md:26]: 4.7.x eol should be 4.8.x release date + 3 months.
```suggestion
| 4.7.x   | Supported | 07-May-2024  | 22-Aug-2024 |
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice start!

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice one! One last thought about the graph and then you're good to go!

File: content/en/core/releases/4.8.0.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+## Highlights
+
+### Performance
+
+The Contacts page loads much faster now. Using [apdex scores and testing on low spec devices](https://forum.communityhealthtoolkit.org/t/evaluating-app-performance-using-apdex-lets-dive-in-and-look-at-an-example/3475), we were able to identify slow areas and make improvements to both the [list view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-list-view/3501) and the [detail view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-detail-view/3486). 
+
+#### Apdex Improvements
+
+{{< figure src="../images/4.8.0-apdex-improvements.png" >}}
```
[the remark this answers is not in the document]
Comment (Benmuiruri) [reply content/en/core/releases/4.8.0.md:34]: Wow! My graph sucked. Yours is fantastic!! 

Thanks for drawing it up. I will use that it as a reference for future graphs

**Review by garethbowen — APPROVED** [review]
Awesome! Thanks.

**Comment by Benmuiruri** [comment]
@mrjones-plip Are you okay with what we have here, I go ahead and merge ?

**Comment by mrjones-plip** [comment]
ship it!!

**Pull request merged**